src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/dto/ActionPlanSessionsDTO.kt

```diff
@@ -1,13 +1,14 @@
 package uk.gov.justice.digital.hmpps.hmppsinterventionsservice.dto
 
+import com.fasterxml.jackson.annotation.JsonProperty
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.ActionPlanSession
 import uk.gov.justice.digital.hmpps.hmppsinterventionsservice.jpa.entity.Attended
 import java.time.OffsetDateTime
 import java.util.UUID
 
 data class UpdateAppointmentDTO(
   val appointmentTime: OffsetDateTime,
-  val durationInMinutes: Int,
+  @JsonProperty(required = true) val durationInMinutes: Int,
```

Comment:
This is needed, otherwise the duration will default to '0' if it isn't included. This will then set the duration of the appointment to 0.

Reply:
is there anywhere else we have not nullable basic types?

Reply:
@lawrence-forooghian can we have a contract test to validate this behaviour?